app/renderer/ui/main-view/main-view.vue

```diff
@@ -9,6 +9,8 @@ import { PaperEntity } from "@/models/paper-entity";
 import { IFeedEntityCollection } from "@/repositories/db-repository/feed-entity-repository";
 import { IPaperEntityCollection } from "@/repositories/db-repository/paper-entity-repository";
 
+import { CategorizerType } from "@/models/categorizer";
```

Comment:
About the importing format.

Please remove the blank line above this and add a blank line below this.